tests/create_LWE.py

```diff
@@ -0,0 +1,242 @@
+from addict import Dict
+
+from CADETProcess.simulator import Cadet
+
+from CADETProcess.processModel import (
+
+    ComponentSystem,
+    FlowSheet,
+
+    Inlet,
+    Outlet,
+    Cstr,
+    GeneralRateModel,
+    TubularReactor,
+    LumpedRateModelWithoutPores,
+    LumpedRateModelWithPores,
+    MCT,
+
+    Linear,
+    Langmuir,
+    LangmuirLDF,
+    BiLangmuir,
+    BiLangmuirLDF,
+    FreundlichLDF,
+    StericMassAction,
+    AntiLangmuir,
+    Spreading,
+    SelfAssociation,
+    BiStericMassAction,
+    MultistateStericMassAction,
+    SimplifiedMultistateStericMassAction,
+    Saska,
+    GeneralizedIonExchange
+)
+
+
+def create_lwe(n_comp=4, unit_type=None, **kwargs) -> None:
```

Comment:
[nitpicking] Since this function can also raise an exception, the correct type to return would be a `NoReturn` (which you would have to import from `typing` first.

Reply:
[suggestion] Have you considered that `create_lwe` should return a fully configured `Process` object?

Reply:
Right now I planned that the create_lwe() method directly runs the simulation as I thought that is what it does in CADET-core? But I can also keep it modular here and just return the process object(s) if that is preferred.